modular_citadel/code/modules/reagents/chemistry/reagents/healing.dm

```diff
@@ -139,13 +161,40 @@
 	if(iscarbon(holder.my_atom))
 		data["injected_vol"] = data["injected_vol"] + passed_data["injected_vol"]
 		passed_data["injected_vol"] = 0
+	update_name()
 	..()
 
 /datum/reagent/synthtissue/on_new(passed_data)
 	if(!passed_data)
 		return ..()
 	if(passed_data["grown_volume"] > data["grown_volume"])
 		data["grown_volume"] = passed_data["grown_volume"]
+	update_name()
 	..()
 
+/datum/reagent/synthtissue/proc/update_name()
+	switch(data["grown_volume"])
+		if(-INFINITY to 50)
+			name = "Young Synthtissue Colony"
+		if(50 to 80)
+			name = "Adolescent Synthtissue Colony"
+		if(80 to 135)
+			name = "Adult Synthtissue Colony"
+		if(135 to 175)
+			name = "Master Synthtissue Colony"
+		if(175 to INFINITY)
+			name = "Apex Synthtissue Colony"
```

Comment:
Wouldn't prefixes like very small / small / average / large / very large make more sense since it's the volume of a colony.

Reply:
I agree, I don't like the names but, I don't want to conflate colony size with volume. I'll use stem cell types instead.